client/landing/stepper/declarative-flow/internals/steps-repository/site-migration-credentials/test/index.tsx

```diff
@@ -101,14 +117,29 @@ const siteInfoUsingWPCOM = {
 	},
 };
 
+const isApplicationPasswordEnabled = isEnabled( 'automated-migration/application-password' );
+const restaureIsApplicationPasswordEnabled = () => {
```

Comment:
I think it would be more simple to have the flag off for the tests and turn it on and off in the new tests.

Reply:
I think both ways work.
But this way avoids tests refactoring in the future, when we turn on the feature flag in production.